man/sd_bus_message_new_method_error.xml

```diff
@@ -0,0 +1,184 @@
+    information from error <parameter>e</parameter> is appended: the
+    <parameter>name</parameter> field of <parameter>e</parameter> is used as the
+    "name" field in the reply header, and and the <parameter>message</parameter>
+    field is appended if present. The error <parameter>e</parameter> must have the
+    <parameter>name</parameter> field set, see
+    <citerefentry><refentrytitle>sd_bus_error_is_set</refentrytitle><manvolnum>3</manvolnum></citerefentry>.
+    </para>
+
+    <para>The <function>sd_bus_message_new_method_errorf()</function> function
+    creates an error reply similarly to
+    <function>sd_bus_message_new_method_error()</function>, but instead of a ready
+    error structure, it takes an error name string <parameter>name</parameter>, plus
+    a <citerefentry
+    project='man-pages'><refentrytitle>printf</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+    format string <parameter>format</parameter> and corresponding arguments. An error
+    reply is sent with the name <parameter>name</parameter> and the formatted string
+    as the message. <parameter>name</parameter> and <parameter>format</parameter>
+    must not be <constant>NULL</constant>.
+    </para>
+
+    <para>The <function>sd_bus_message_new_method_errno()</function> function creates
+    an error reply similarly to
+    <function>sd_bus_message_new_method_error()</function>, but in addition to the
+    error structure <parameter>p</parameter>, it takes an
+    <citerefentry><refentrytitle>errno</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+    error value in parameter <parameter>error</parameter>. If the error
+    <parameter>p</parameter> is set (see
+    <citerefentry><refentrytitle>sd_bus_error_is_set</refentrytitle><manvolnum>3</manvolnum></citerefentry>),
+    it is used in the reply. Otherwise, <parameter>error</parameter> is used to
+    create a new error structure using
+    <citerefentry><refentrytitle>sd_bus_error_set_errno</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+    and that is used in the reply. (If <parameter>error</parameter> is zero, no error
+    is actually set, and an error reply with no information is created.)</para>
+
+    <para>The <function>sd_bus_message_new_method_errnof()</function> function
+    creates an error reply similarly to
+    <function>sd_bus_message_new_method_error()</function>. It takes an
+    <citerefentry><refentrytitle>errno</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+    error value in parameter <parameter>error</parameter>, plus a
+    <citerefentry project='man-pages'><refentrytitle>printf</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+    format string <parameter>format</parameter> and corresponding arguments. The
+    error message is initalized using the error value and formatted string. (If
+    <parameter>error</parameter> is zero, no error is actually set, and an error
+    reply with no information is created.)</para>
```

Comment:
this needs to say that the error string corresponding to the specified errro code is available as `%m` in the format string